api/serializers/uqj.py

```diff
@@ -37,4 +37,4 @@ class Meta:
         fields = ['id', 'last_viewed', 'opened_tutorial', 'tokens_received', 'is_solved', 'is_partially_solved',
                   'question', 'question_id', 'num_attempts', 'status', 'formatted_current_tokens_received',
                   'is_allowed_to_submit', 'variables', 'variables_errors', 'rendered_text', 'rendered_choices',
-                  'rendered_lines', 'status_class', 'input_files', 'is_checkbox']
+                  'rendered_lines', 'status_class', 'input_files', 'is_checkbox', 'is_favourite']
```

Comment:
Each PR should only include the files related to the feature. Remove the unnecessary/duplicate changes from other features.

Reply:
Good call. I just branched off of theirs. I will remove it!